wa/workloads/gmail/__init__.py

```diff
@@ -36,6 +36,12 @@ class Gmail(ApkUiautoWorkload):
     6. Enter text in the Compose field
     7. Click the Send mail button
 
+    To run the workload in offline mode, a 'mailstore.tar' file is required. In order to
+    generate such a file, Gmail should first be operated from an Internet-connected environment.
+    After this, the relevant database files can be found in the
+    '/data/data/com.google.android.gm/databases' directory. These files can then be archived to
+    produce a tarball using a command such as ``tar -cvf mailstore.tar -C /path/to/databases .``.
+
```

Comment:
Should also add that the file should be dropped into '~/.workload_automation/dependencies/gmail/` (creating that directory if it doesn't already  exist.